django-stubs/contrib/staticfiles/storage.pyi

```diff
@@ -10,8 +10,6 @@ from typing_extensions import TypeAlias
 _PostProcessT: TypeAlias = Iterator[tuple[str, str, bool] | tuple[str, None, RuntimeError]]
 
 class StaticFilesStorage(FileSystemStorage):
-    base_location: str
-    location: _PathCompatible
```

Comment:
ALready defined correctly in inherited `FileSystemStorage`

Reply:
Hrm, looking at `StaticFilesStorage.__init__`, these could be `cached_property` **or** instance attribute `None` (if `settings.STATIC_ROOT` is unset). I'm not sure that can be typehinted accurately.

But we should add `None` to possible values, up to you if you want to do it in this PR.

Reply:
Django [will raise](https://github.com/django/django/blob/main/django/contrib/staticfiles/storage.py#L37-L44) if `self.location` is not set so I don't know if we should add None t possible values ?